docs/docs/guides/adding-custom-metrics-table-calculations-explore.mdx

```diff
@@ -0,0 +1,254 @@
+
+```sql
+${pages.page_view_count} * 100 / SUM(${pages.page_view_count}) OVER()
+```
+
+In general, the SQL used for calculating the percent of the total has just one important column:
+
+- `column_i_want_to_see_the_percent_of_total` - this is the column that you want to see each value's percent of the total values in that column
+
+So, the SQL for the percent of a total will always look something like:
+
+```sql
+${table.column_i_want_to_see_the_percent_of_total} * 100 / SUM(${table.column_i_want_to_see_the_percent_of_total}) OVER()
+```
+
+**Level up your SQL**:
+
+- You can round your table calculation to only show a certain number of decimal points. Here's an example where I only show two decimal points:
+
+  ```sql
+  ROUND(${table.column_i_want_to_see_the_percent_of_total} * 100 / SUM(${table.  column_i_want_to_see_the_percent_of_total}) OVER(), 2)
+  ```
+
+<img src={PercentOfTotalRounded} width="320" height="388" style={{display: "block", margin: "0 auto 20px auto"}}/>
+
+#### Percent of previous
+
+The percent of previous gives you the percent of a value compared to the value in the row above it.
+
+Here's an example of a percent of previous calculation:
+
+<img src={PercentOfPrevious} width="1227" height="473" style={{display: "block", margin: "0 auto 20px auto"}}/>
+
+And here's the SQL that was used to generate it:
+
+```sql
+${pages.page_view_count} * 100 / LAG(${pages.page_view_count}) OVER(ORDER BY ${pages.date_day}, ${pages.path})
+```
+
+In general, the SQL used for calculating the percent of the previous value has two bits (with an optional third bit):
+
+- `column_I_want_to_compare` - this is the column with the values you want to compare
+- `column_I_want_to_order_by` - this is the column you want to use to order the values you want to compare
+- `optional_other_column_I_want_to_order_by` - this column is optional and you can add as many more `order by` columns as you need. Normally, you'll need to add every dimension in your results table to the `ORDER BY` bit in your SQL. And, the order of these will need to be the same as the ordering you've added to the columns in your results table.
+
+So, the SQL for the percent of previous will always look something like:
+
+```sql
+${table.column_i_want_to_compare} * 100 / LAG(${table.column_i_want_to_compare}) OVER(ORDER BY ${table.column_I_want_to_order_by}, ${table.optional_other_column_I_want_to_order_by})
+```
+
+**Level up your SQL**:
+
+- You can round your table calculation to only show a certain number of decimal points. Here's an example where I only show two decimal points:
+
+  ```sql
+  ROUND(${table.column_i_want_to_compare} * 100 / LAG(${table.column_i_want_to_compare}) OVER(ORDER BY ${table.column_I_want_to_order_by}, ${table.optional_other_column_I_want_to_order_by}), 2)
+  ```
+
+<img src={PercentOfTotalRounded} width="320" height="388" style={{display: "block", margin: "0 auto 20px auto"}}/>
+
+#### Percent change from previous
```

Comment:
I find these titles small and hard to find in a wall of text 🔍 👀 Wondering if we should bump the template titles one level up.

Reply:
yes! already did this too once I started looking at it :)